app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -84,10 +84,14 @@ def container_class(application_choice)
       return unless @editable
 
       if application_choice.course_option_availability_error?
-        "govuk-inset-text app-inset-text--narrow-border app-inset-text--#{@application_choice_error ? 'error' : 'important'}"
+        "govuk-inset-text app-inset-text--narrow-border app-inset-text--#{@application_choice_error ? 'error' : 'important'} govuk-!-padding-top-0 govuk-!-padding-bottom-0"
       end
     end
 
+    def course_not_open_for_applications_yet?(application_choice)
+      application_choice.course.applications_open_from > Time.zone.today
```

Comment:
probably this warrants a method on `Course`, corresponding to the `open_for_applications` scope